src/app/navbar/navbar.component.ts

```diff
@@ -58,13 +71,25 @@ export class NavbarComponent extends Logout implements OnInit {
       .subscribe(() => {
         this.isExtended = toExtendSite(this.router.url);
       });
+    this.isAdminOrCurator$ = this.userQuery.isAdminOrCurator$;
   }
 
   ngOnInit() {
     this.userQuery.user$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((user) => {
       this.user = user;
+      this.requestsService.updateMyMemberships();
+      this.requestsService.updateCuratorOrganizations();
     });
     this.userQuery.extendedUserData$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((extendedUser) => (this.extendedUser = extendedUser));
+    this.myOrganizationInvites$ = this.requestsQuery.myOrganizationInvites$;
+    this.myRejectedOrganizationRequests$ = this.requestsQuery.myRejectedOrganizationRequests$;
+    this.allPendingOrganizations$ = this.requestsQuery.allPendingOrganizations$;
+    combineLatest([this.myOrganizationInvites$, this.myRejectedOrganizationRequests$, this.allPendingOrganizations$]).subscribe(
+      ([invites, rejections, pendingOrgs]: [Array<OrganizationUser>, Array<OrganizationUser>, Array<Organization>]) => {
+        this.adminNotifCount = invites?.length + rejections?.length + pendingOrgs?.length;
+        this.notifCount = invites?.length + rejections?.length;
```

Comment:
Instead of having `adminNotifCount` and `notifCount`, you could have just `notifCount` and add `pendingOrgs?.length` to `notifCount` if the user is an admin. This way, your HTML is simplified because you don't have to check if the user is an admin to switch between `adminNotifCount` and `notifCount`